addons/udes_stock/models/stock_quant.py

```diff
@@ -153,6 +153,18 @@ def get_move_lines(self, aux_domain=None):
         """
         self.ensure_one()
         MoveLine = self.env["stock.move.line"]
+        domain = self.get_move_lines_domain(aux_domain=aux_domain)
+        return MoveLine.search(domain)
+
+    def get_move_lines_domain(self, aux_domain=None):
+        """
+        Getting the domain in its own method, would be easier to reuse it when we need just domain
+        and example to execute search_read method.
+        It can be easier even if is needed to extend with super it without having to extend with
```

Comment:
"extend it wit super" is more idiomatic English.

Reply:
You mean "with" :)